Svc/FrameAccumulator/FrameAccumulator.cpp

```diff
@@ -0,0 +1,166 @@
+// ======================================================================
```

Comment:
Might need to reimport this code, if and when my PR merges that changes types.

Reply:
Yes, I'm blocking on https://github.com/nasa/fprime/pull/3286 to be merged so that CI here can pass